src/coreComponents/common/Logger.hpp

```diff
@@ -490,7 +490,7 @@ struct InputError : public std::runtime_error
   /**
    * @brief Returns an exception message in which we insert another message.
    */
-  static std::string InsertExMsg( std::string const & originalMsg, std::string const & msgToInsert );
+  std::string InsertExMsg( std::string const & originalMsg, std::string const & msgToInsert );
```

Comment:
Where do you use `this` in this member function?
You can make it a free function only in the `cpp` file, or keep it `private static` if you want.
But I'm not sure to understand making it a member function.

Reply:
I misunderstood the previous input of @untereiner. I'll change it to a free function in the `cpp`.